specs/protocol/isthmus/derivation.md

```diff
@@ -69,13 +69,20 @@ cast keccak $(cast concat-hex 0x000000000000000000000000000000000000000000000000
 Verify `data`:
 
 ```bash
-git checkout TODO
+git checkout 9436dba8c4c906e36675f5922e57d1b55582889e
 make build-contracts
 jq -r ".bytecode.object" packages/contracts-bedrock/forge-artifacts/L1Block.sol/L1Block.json
 ```
 
 This transaction MUST deploy a contract with the following code hash
-`TODO`.
+0x8e3fe7a416d3e5f3b7be74ddd4e7e58e516fa3f80b67c6d930e3cd7297da4a4b.
+
+To verify the code hash:
```

Comment:
Oops ci lints are failing because there's no empty line between this line and the code block.

Just need to insert a line here.

```suggestion
To verify the code hash:

```